django_freeradius/tests/test_api.py

```diff
@@ -2,8 +2,8 @@
 
 import swapper
 from django.contrib.auth.models import User
-from django.core.urlresolvers import reverse
 from django.test import TestCase
+from django.urls import reverse
```

Comment:
will this work on both django 1.11 and 2.0? since we have to keep supporting both

Reply:
in my local env it did well

Reply:
yes, `django.url.reverse` is available since django 1.11 or 1.10